src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -426,9 +427,71 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
     this.router.navigate(['logs/syslog'], { queryParams: { source: service.name } });
   }
 
-  navToServiceConfiguration(service){
-    let routePath = service.type === 'Northbound' ? 'north' : 'south';
-    this.router.navigate([routePath, service.name, 'details'])
+  public showConfigWithPollingSchedule(serviceInfo): void {
+    this.ngProgress.start();
+    this.schedulesService.getSchedules().
+      subscribe(
+        (data: any) => {
+          this.ngProgress.done();
+          const pollingScheduleID = data.schedules.find(s => s.processName === 'manage')?.id;
+          serviceInfo['pollingScheduleID'] = pollingScheduleID;
+          this.router.navigate(['/developer/options/additional-services/config'], { queryParams: { ...serviceInfo },  skipLocationChange: true });
+        },
+        error => {
+          this.ngProgress.done();
+          if (error.status === 0) {
+            console.log('service down ', error);
+          } else {
+            this.alertService.error(error.statusText);
+          }
+        });
+  }
+
+  public isDeveloperFeatureOn(): boolean {
+    const devFeature = JSON.parse(localStorage.getItem('DEV_FEATURES'));
+    return devFeature ? devFeature : false;
   }
-}
 
+  navToServiceConfiguration(service) {
+    let serviceInfo = {
+      name: service.name,
+      isEnabled: service.status === 'running' ? true : false,
+      added: true,
+      isInstalled: true
+    }
+    switch (service.type) {
+      case 'Northbound':
+        this.router.navigate(['north', service.name, 'details']);
+        break;
+      case 'Southbound':
+        this.router.navigate(['south', service.name, 'details']);
+        break;
+      case 'BucketStorage':
+        serviceInfo['process'] = 'bucket';
+        serviceInfo['type'] = 'BucketStorage';
+        serviceInfo['package'] = 'fledge-service-bucket';
+        this.router.navigate(['/developer/options/additional-services/config'], { queryParams: { ...serviceInfo },  skipLocationChange: true });
```

Comment:
Sending big object in the queryParams not looks a good idea to me
```
{
    "name": "N1",
    "isEnabled": "true",
    "added": "true",
    "isInstalled": "true",
    "process": "notification",
    "type": "Notification",
    "package": "fledge-service-notification"
}
```

And why we need to do `skipLocationChange: true`? To hide the object in the queryParam? 
By doing this the path `/developer/options/additional-services/config` will not be visible in the browser Address bar ever.

There must be better way to do this.